docs/dasharo-tools-suite/documentation/features.md

```diff
@@ -270,19 +13,42 @@ This section describes the functionality of the Dasharo Tools Suite. These are:
     + [run commands from iPXE shell automatically](#run-commands-from-ipxe-shell-automatically),
     + [run DTS using VentoyOS](#run-dts-using-ventoyos).
 
-### Dasharo zero-touch initial deployment
+## Available Commands
+
+When DTS is started, it has following options for the user to choose from:
+
+* **(1)** Dasharo HCL Report:
+
+  Generate HCL report on the current machine.
+  Read more: [HCLReport](#hcl-report)
+
+* **(2)** Update Dasharo Firmware
+* **(3)** Restore Firmware from Dasharo HCL Report
+* **(4)** Load your DPP keys
+
+Load your Dasharo Pro Package (DPP) keys. Read more:
+[How can I use my Dasharo Pro Package credentials](../../osf-trivia-list/dts.md#how-can-i-use-my-dasharo-pro-package-credentials)
+
+* **(R)** Reboot
+* **(P)** Poweroff
+* **(S)** Enter shell
+* **(K)** Launch SSH Server
+* **(L)** Enable sending DTS logs
+* **(V)** Enable verbose mode
```

Comment:
```suggestion* **1)** [Dasharo HCL Report](#hcl-report)
* **2)** [Update Dasharo Firmware](#firmware-update) or [Install Dasharo Firmware](#dasharo-zero-touch-initial-deployment)
* **3)** [Restore Firmware from Dasharo HCL Report](#update-issues)
* **4)** [Load your DPP keys](../../osf-trivia-list/dts.md#how-can-i-use-my-dasharo-pro-package-credentials)
    \- Load your Dasharo Pro Package (DPP) keys
* **R** Reboot
* **P** Poweroff
* **S** Enter shell
* **K** Launch SSH Server
* **L** [Enable sending DTS logs](../../osf-trivia-list/dts.md#how-can-i-help-the-support-team-diagnose-my-problem-faster)
* **V** [Enable verbose mode](../../osf-trivia-list/dts.md#how-can-i-help-the-support-team-diagnose-my-problem-faster)
```

Maybe like this?

Reply:
@DaniilKl there is also [5) DTS extensions](https://github.com/Dasharo/dts-scripts/blob/c7b8d036f549c41f848ee254ba309baf762835d5/include/dts-functions.sh#L1346), is it already available and if yes can you help to expand this docs or point to documentation?

Reply:
Introduced this style in: b133540c400db01df4c8bcaf581f2103c23cd870